subgraphs/tokemak/src/common/tokens.ts

```diff
@@ -26,17 +26,19 @@ export function getOrCreateToken(address: Address): Token {
 export function getOrCreateRewardToken(address: Address): RewardToken {
   let id = address.toHexString();
   let token = RewardToken.load(id);
-  if (!token) {
+  if (!token || token.name === "") {
```

Comment:
why do we need `|| token.name === ""` if we are already hardcoding name and symbol?

Reply:
For new vaults